target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -9,6 +9,9 @@ board=$(board_name)
 case "$FIRMWARE" in
 "ath9k-eeprom-ahb-18100000.wmac.bin")
 	case $board in
+	8dev,lima)
+		caldata_extract "art" 0x1000 0x800
+		;;
```

Comment:
This can be merged with ubnt,unifi (i.e. the latter should be moved up here, too).

Reply:
Merged.